docs/constants-and-vars.rst

```diff
@@ -28,7 +29,11 @@ Name                 Type             Value
 
 .. note::
 
-    ``msg.sender`` and ``msg.value`` can only be accessed from external functions. If you require these values within a private function they must be passed as parameters.
+    ``msg.data``, ``msg.sender`` and ``msg.value`` can only be accessed from external functions. If you require these values within a private function they must be passed as parameters.
+
+.. note::
+
+    ``msg.data`` requires the usage of :func:`slice <slice>` to explicitly extract a section of calldata. If the extracted section exceeds the bounds of calldata, this will throw. You can check the size of ``msg.data`` using :func:`len <len>`.   
```

Comment:
This check is not implemented, correct?

```suggestion
    ``msg.data`` requires the usage of :func:`slice <slice>` to explicitly extract a section of calldata. You can check the size of ``msg.data`` using :func:`len <len>`.   
```

Reply:
Yes it's just one line, should I take it out ? :thinking: